### DISCUSSION TITLE
Lotta-Layouts (Vorlagen) mit originellen Namen (Bezug zur Schule) bezeichnen.

### DISCUSSION THREAD

**Discussion by wie0815** [lotta-schule/web#285] [Ideen]
Damit es nicht in Vergessenheit gerät.
<img width="1146" alt="Bildschirmfoto 2024-07-26 um 10 17 04" src="https://github.com/user-attachments/assets/bee977f0-2bf9-4f51-8622-792c77b74cda">
Beispiele
statt Standard: "Bildungseinrichtung"
statt Purple Pastell: "Klassenkasper"
statt Neutral: "Klassenbester"
satt Retro Contrast: "Schiefertafel"

**Comment by wie0815** [comment]
Vielleicht schon ab Version 5.0?

**Reply by ptitmouton** [reply]
Ja das sollte recht schnell gemacht sein denke ich

**Comment by ptitmouton** [comment]
Ticket #290 dazu erstellt.

Ich denke die Diskussion kann damit erstmal geschlossen werden.
